packages/expect/src/__tests__/matchers.test.js

```diff
@@ -606,6 +606,56 @@ describe('.toEqual()', () => {
     });
     expect(actual).toEqual({x: 3});
   });
+
+  describe('cyclic object equality', () => {
+    test('properties with the same circularity are equal', () => {
+      const a = {};
+      a.x = a;
+      const b = {};
+      b.x = b;
+      expect(a).toEqual(b);
```

Comment:
To make explicit guarantee that the comparison is symmetrical, do we want to double up on all the assertions?

Like the pair in the last test case.

Reply:
Fair point.
I will add another assertion to the test cases in which cyclic objects have the same shape.

Reply:
Let’s be explicit both for positive `.toEqual` or negative `.not.toEqual` There have been multiple new issues this week with inconsistent asymmetrical negative assertions in `expect` package.